kolibri_explore_plugin/test/channels/README.md

```diff
@@ -0,0 +1,21 @@
+# Test channel DB data
+
+Each JSON file represents a Kolibri channel in a format that can be used
```

Comment:
Hmm, my main motivation was to avoid committing a bunch of sqlite databases to the repo, but that would be the logical next step. It would certainly be possible and you could skip the intermediate JSON step since it's really just a storage format to feed data into sqlalchemy to create the databases. I wouldn't be that concerned about the consistency as there's only a couple spots where the IDs are used in the tests and they could just as easily be introspected from the generated data.

That said, I'm not that keen to sink more time into this issue. The nice thing about having the JSON statically files in tree is that you can inspect them at your leisure if needed or even construct a manual test with them. I'll take a look at doing this tomorrow, but if it seems like a big time suck I'll punt.

Reply:
I'm going to punt on this. The other thing that would be nice about generating everything on the fly is that if you make any changes to the template files, then you don't have to update/regenerate the test files. However, I don't think I have any time left to sink into this. I leave it as an exercise for a future maintainer :smile:

Reply:
Agreed, let's move forward.